src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,38 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+
+export const PostInfo = ({ posts, users, comments }) => (
```

Comment:
```suggestion
export const PostInfo = ({ post }) => (
```
pass only post prop here, the user info and comments array should be as the field in this post object